style.css

```diff
@@ -385,3 +385,71 @@ ul {
 .langag-btn .lang-body {
   display: flex;
 }
+
+/* Footer */
+
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background: url("./images/illustration-contact-form.svg") no-repeat;
+  background-size: 10%;
+  background-position: bottom 275px right 0;
+  width: 100%;
+}
+
+.contact-msg {
+  font-weight: bold;
+  font-size: 40px;
+  text-align: center;
+  line-height: 52px;
+  color: #172b4d;
+}
+
+.form_items {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.form-box {
+  width: 327px;
+  border: 1px solid #d0d9d4;
+  border-radius: 4px;
+  padding: 15px 16px;
+  font-size: 15px;
+}
+
+.form_items li {
+  margin: 10px auto;
+}
+
+.form_items textarea {
+  background-color: #fff;
+  height: 114px;
+  width: 327px;
+  border: 1px solid #60c095;
+  line-height: 24px;
+  padding: 12px;
+  font-size: 16px;
+  border-radius: 4px;
+}
+
+.contact-form .email-link {
+  color: #42526e;
+  font-size: 20px;
+  line-height: 32px;
+  text-align: center;
+  margin-bottom: 147px;
+  box-sizing: border-box;
+}
+
+::placeholder {
+  color: #091e42;
+}
+
+.contact-form input:focus,
+.contact-form textarea:focus {
+  border: 2px solid #091e42;
+  outline: none;
+}
```

Comment:
- [ ] [OPTIONAL]  I would kindly suggest to modify the `texarea` `placeholder` text style to match it with Figma design: 

Your design  | Figma design
------------- | -------------
![image](https://user-images.githubusercontent.com/77038610/144088912-675c8ba7-960f-4a8f-b9f0-e7453439b46f.png)  | ![image](https://user-images.githubusercontent.com/77038610/144088936-ea7dcd25-c4d5-40b7-98e8-ac5ac8738e01.png)

✔️ Here is how you can target `textarea` `placeholder` in css `textarea::placeholder`, read more about pseudo selectors [here](https://developer.mozilla.org/en-US/docs/Web/CSS/::placeholder)